package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/fusion-components",
-    "version": "0.1.78",
+    "version": "0.1.79",
```

Comment:
We shouldn't bump version in pull requests. Can result in confusions if multiplie PRs bumps to different versions and are merged in the wrong order